Source/WebCore/page/Quirks.cpp

```diff
@@ -1478,7 +1478,7 @@ bool Quirks::allowLayeredFullscreenVideos() const
 }
 #endif
 
-#if PLATFORM(VISION)
+#if ENABLE(VIDEO_USES_ELEMENT_FULLSCREEN)
 // twitter.com: rdar://124180748
 bool Quirks::shouldDisableFullscreenVideoAspectRatioAdaptiveSizing() const
```

Comment:
I don't think it really makes sense to change this one, and could instead be guarded at the call site.